### ISSUE TITLE
Cannot parse SQL queries using functions

### ISSUE THREAD

**Issue by oomathias** [sbdchd/squawk#55]
Hi, thanks for creating this tool.

I have a few migrations that cannot be parsed when using functions.

```sql
ALTER TABLE foobar ALTER COLUMN value SET DEFAULT TO_JSON(false);
```

```sh
Err(
    CheckSQL(
        ParsingSQL(
            JsonParse(
                "unknown variant `FuncCall`, expected one of `Constraint`, `ColumnDef`, `A_Const`, `ReplicaIdentityStmt` at line 1 column 220",
            ),
        ),
    ),
)
```

Do you plan to support those?

**Comment by sbdchd** [comment]
Yeah shouldn't be too hard.

Currently squawk parses the json from https://github.com/lfittl/libpg_query into structures to make writing the lints less painful. Seems I missed some cases.

**Comment by oomathias** [comment]
Thanks for being super reactive.

**Comment by sbdchd** [comment]
I need to futz with the releases a bit before it's available via `curl` or `npm`, thanks for the report

**Comment by sbdchd** [comment]
I've released the new version to `npm` and the binaries are also available attached to the GitHub release
